lib/influxer/metrics/metrics.rb

```diff
@@ -45,6 +46,12 @@ def attributes(*attrs)
         end
       end
 
+      def values(*attrs)
```

Comment:
Yes, i make this change for less complexity on values method, but i can change. What do you think?